Assets/Scripts/LifeSystem/DeathBarierSystem.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DeathBarierSystem : MonoBehaviour
+{
+    private LifeSystem _lifeSystem;
+    private void Start()
+    {
+        _lifeSystem = LifeSystem.Instance;
+    }
+    private void OnTriggerEnter(Collider other)
+    {
+        if (other.CompareTag("Player"))
+        {
+            _lifeSystem.SubtrackLife();
+            _lifeSystem.RespawnPlayer();
+            Destroy(other);
+        }
+    }
+}
```

Comment:
```suggestion
```
We zullen Uiteindelijk een script hebben voor de death barrier die een event fired en de player "respawned" . we hoeven dan aleen in unity in de event SubtrackLife aanroepen. de rest is niet nodig no offfence natuurlijk 😅.